### DISCUSSION TITLE
Is it possible to make feed update interval time even lower value

### DISCUSSION THREAD

**Discussion by Arifmahamud120** [tughi/aggregator-android#45] [General]
Hello
Aggregator Next Team

I have a feature request that, if it's possible please make feed update interval time even lower value than 15 minutes

I would like, if i can update feeds in every 5 second. 

Thanks

**Comment by ghost** [comment]
It's not team per se. The only dev is tughi. 
Even if it's possible to refresh every often & so, it'll be a background battery hogging nightmare.
It's better to have 15 minutes as the least, caz that itself refreshes content like 96 times a day.

**Comment by tughi** [comment]
A lower update interval would drain your battery in a couple of hours.
